apis/github

```diff
@@ -17,6 +16,42 @@ local function getAPI(path)
 	end
 end
 
+-- A class for authorization
+local authFile = '.github-auth'
+local function writeAuth(data)
```

Comment:
Don't see why this should be a free function. Why not just leave it in `Auth.save`?